bft/dialects/types.py

```diff
@@ -120,8 +120,6 @@ def mapping_for_case(self, case: Case) -> SqlMapping:
         dfunc_scalar = self.__scalar_functions_by_name.get(case.function, None)
         dfunc_aggregate = self.__aggregate_functions_by_name.get(case.function, None)
         dfunc = dfunc_scalar or dfunc_aggregate
-        if dfunc.unsupported:
```

Comment:
Can we keep this unsupported check somehow?  We still want to skip these tests from running with pytest.

Reply:
This is kind of a hacky way to get around this issue:
https://github.com/richtia/bft2/blob/c9d9e79a5430872ccafe35a8015412020edc553b/bft/dialects/types.py#L124-L131

Reply:
Actually a better way would be to do something like this
```
        if "PYTEST_CURRENT_TEST" not in os.environ:
            if dfunc is None:
                return None
        elif dfunc.unsupported:
            pytest.skip("Skipping unsupported function.")
```
make sure to import os

Reply:
made the change, thanks!